components/cache/adapters/redis_adapter.rst

```diff
@@ -258,6 +258,7 @@ Read more about this topic in the official `Redis LRU Cache Documentation`_.
 .. _`Redis`: https://github.com/phpredis/phpredis
 .. _`RedisArray`: https://github.com/phpredis/phpredis/blob/master/arrays.markdown#readme
 .. _`RedisCluster`: https://github.com/phpredis/phpredis/blob/master/cluster.markdown#readme
+.. _`Relay`: https://relay.so/
```

Comment:
Might be nice to link to GitHub instead:

```suggestion
-.. _`Relay`: https://relay.so/
+.. _`Relay`: https://github.com/cachewerk/relay
```

Reply:
I don't really agree. If you want to go to documentation, GH repo will redirect you to relay.so anyways. I know you are following other, above listed clients, but none of them have a project website.

Reply:
Ah, right. Maybe `https://relay.so/docs`?

Reply:
Let's use the project website 👍